src/history/hs.c

```diff
@@ -1286,6 +1288,8 @@ __verify_history_store_id(WT_SESSION_IMPL *session, WT_CURSOR_BTREE *cbt, uint32
         WT_WITH_PAGE_INDEX(session, ret = __wt_row_search(cbt, hs_key, false, NULL, false, NULL));
         WT_ERR(ret);
 
+/* FIXME: temporarily disable hs verification. */
+#if 0
```

Comment:
It seems like this would be a more localized change with just:
```
#if 0
    found = cbt->compare == 0;
#else 
    found = true;
#endif

Reply:
Changed